app/dashboard/trash/TrashBinIndex.tsx

```
@@ -0,0 +1,31 @@
+      {trashTasks &&
+        trashTasks.map((task) => (
+          <TaskCard
+            id={task.id}
+            name={task.name}
+            key={task.id}
+            completedAt={task.completedAt}
+            createdByOptimisticUpdate={task.createdByOptimisticUpdate}
+            description={task.description}
+            tags={task.tags}
+          />
+        ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)